Sources/Core/FileGenerator.swift

```diff
@@ -19,6 +19,7 @@ public enum GenerationParameterType {
     case includeRuntime
     case indent
     case packageName
+    case nullabilityAnnotationType
```

Comment:
Since this is language specific, can you prefix with `java`?

Reply:
Thanks for the clarification. I was wondering if it was a convention to strip the language from the variable name here, because `packageName` is also Java-specific.